cmd/lhsm-plugin-az-core/archive.go

```diff
@@ -50,5 +109,45 @@ func Archive(o ArchiveOptions) (int64, error){
 			Metadata:    meta,
 		})
 
+	if err != nil {
+		return 0, err
+	}
+
+	if hnsEnabledAccount {
+		aclResp, err := blobURL.GetAccessControl(ctx, nil, nil, nil, nil, nil, nil, nil, nil)
```

Comment:
This GetAccessControl needs to happen before the upload, otherwise the acl field would have been lost already.